database/example.config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "MYSQL-HOST": "localhost",
+    "MYSQL-USER": "root",
+    "MYSQL-PASS": "password",
+    "MYSQL-DB": "boilerbooks"
+}
```

Comment:
maybe, I figured this is more 1 time use so it was okay. But probably could get merged somehow